docs/vercel_requirements.txt

```diff
@@ -2,3 +2,4 @@
 -e ../libs/community
 -e ../libs/core
 urllib3==1.26.18
+nbformat
```

Comment:
I don't think this is needed at build time, so doesn't need to be in vercel_requirements right?